spec/requests/users_spec.rb

```diff
@@ -18,11 +17,17 @@
   end
 
   describe 'GET /show' do
-    pending 'not implemented yet'
     it 'returns http success' do
-      get '/users/show'
+      user = create(:user)
+      get "/users/#{user.id}"
```

Comment:
O rails te gera alguns helpers que você pode usar até mesmo aqui nos testes. 

```suggestion
      get user_path, id: user.id
```

Funciona? Não lembro de cabeça agora a sintaxe

Reply:
fiz user_path(user) e deu certo!